mne/coreg.py

```diff
@@ -1963,3 +1969,57 @@ def reset(self):
         self._extra_points_filter = None
         self._update_nearest_calc()
         return self
+
+    def _get_fiducials_distance(self):
+        distance = OrderedDict()
```

Comment:
No longer need `OrderedDict` because we are 3.7+. I think our existing ones are mostly there for cruft reasons